.changeset/large-plants-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"replayio": patch
+---
+
+Fixed a potential data corruption issue related to the `recordings.log` updates
```

Comment:
Since this file now comes in a separate PR, it's good to add the metadata related to the original PR:
```suggestion
pr: #508
commit: e0ce74f

Fixed a potential data corruption issue related to the `recordings.log` updates
```